game/templates/game/delete_account.html

```
@@ -0,0 +1,236 @@
+            <form method="POST">
+
+                {% csrf_token %}
+
+                <div class="form-group">
+
+                    <label>
+                        Username
+                    </label>
+
+                    <input
+                        type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)